README.md

```diff
@@ -158,7 +158,7 @@ Most development discussion is taking place on github in this repo. Further, the
 
 All contributions, bug reports, bug fixes, documentation improvements, enhancements and ideas are welcome.
 
-A detailed overview on how to contribute can be found in the **[contributing guide](https://dev.pandas.io/docs/contributing.html)**. There is also an [overview](.github/CONTRIBUTING.md) on GitHub.
+A detailed overview on how to contribute can be found in the **[contributing guide](https://pandas.io/docs/development/contributing.html)**. There is also an [overview](.github/CONTRIBUTING.md) on GitHub.
```

Comment:
Can you leave the `dev.` for now? (until we have a proper place for the dev docs) The link you are changing it to now are the stable docs

Reply:
> The link you are changing it to now are the stable docs

see https://github.com/pandas-dev/pandas/pull/32068#issuecomment-593362224

dev docs are stale

![image](https://user-images.githubusercontent.com/13159005/75776329-b2af8880-5d4b-11ea-86ab-5bd2961da092.png)


this new link is not the stable docs

![image](https://user-images.githubusercontent.com/13159005/75776288-9e6b8b80-5d4b-11ea-8a49-068ba88063bf.png)